Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:22.04
+
+# docker build -t oras-py .
+
+LABEL MAINTAINER @vsoch
+ENV PATH /opt/conda/bin:${PATH}
+ENV LANG C.UTF-8
+RUN apt-get update && \
+    apt-get install -y wget && \
```

Comment:
coming from the mamba world I'd advocate for the `micromamba` docker image here, and using the `python` packages from the `conda-forge` channel: https://github.com/mamba-org/micromamba-docker/ – you could also install `ipython` from conda-forge :)

Reply:
Haha, fair! We could probably add another recipe for that if you think it’s much better, even elsewhere. My preference generally is for more vanilla base images but you are right I broke that rule a bit with conda instead of installing python3. We can have a discussion with other maintainers to gauge preferences - I’d be ok with any of these ideas really!

Reply:
I guess if you want to use `pip` anyways, we could also use the official `python` base images? (https://hub.docker.com/_/python)